modules/meeting/app/models/queries/meetings/meeting_query.rb

```diff
@@ -41,7 +41,9 @@ def results
     end
 
     def default_scope
-      Meeting.visible(user)
+      Meeting
+        .visible(user)
+        .unscope(:order)
```

Comment:
Hi @oliverguenther 
I have a question: .unscope(:order): removes any pre-existing ORDER BY clauses, does it?
If we want to order by date ascending, can we write sth like this:
```
   .unscope(:order)
    .order(date: :asc)
```
it adds an explicit ordering to ensure meetings are sorted by date in ascending order, is it correct?